skimage/filters/thresholding.py

```diff
@@ -1105,10 +1109,6 @@ def threshold_multiotsu(image, classes=3, nbins=256):
 
     Notes
     -----
-    The threshold values are chosen in a way that maximizes the variance
-    between the desired classes. Based on the Multi-Otsu approach by
-    Liao, Chen and Chung.
-
     This implementation relies on a Cython function whose complexity
     if :math:`O\left(\frac{Ch^{C-1}}{(C-1)!}\right)`, where :math:`h`
```

Comment:
```suggestion
    is :math:`O\left(\frac{Ch^{C-1}}{(C-1)!}\right)`, where :math:`h`
```
Typo? (Strangely GitHub shows a change in whitespace which shouldn't be there. The indent stays at 4 spaces...)

Reply:
@lagru I'm sorry I can't figure out what you've changed here?

Reply:
No problem. :) I only changed the first word: `if` to `is`.

Reply:
Oh! LOL. I was looking at the terrible latex highlighting since you'd convinced me not to look at the left part! :sweat_smile: 

Anyway, I've edited locally to make sure everything is correct. Thanks!